app/views/pets/index.html.erb

```diff
@@ -1,14 +1,21 @@
 <div class="content">
   <h1>
     Pets
-    <%= '(adoptable pets)' if @adoptable %>
-    <%= '(pets that are pending adoption)' if @pending %>
+    <% if (params[:adoptable] == 'true') %>
+      <% pets = @pets.adoptable %>
+      (adoptable pets)
+    <% elsif (params[:adoptable] == 'false') %>
+      <% pets = @pets.pending %>
+      (pets pending adoption)
+    <% else %>
+      <% pets = @pets.sort_by_status %>
+    <% end %>
   </h1>
   <p>
     <span class="spaced"><%= link_to('Show adoptable pets only', '/pets?adoptable=true') %></span>
     <%= link_to('Show pending pets only', '/pets?adoptable=false') %>
   </p>
-  <% @pets.each do |pet| %>
+  <% pets.each do |pet| %>
```

Comment:
Controller exposes Pet.all, view decides if it needs to narrow it down to only adoptable/pending pets, otherwise sorts the collection by status. The if statement conditions are checking params[:adoptable], which comes from the query string when one of those links is clicked. not sure if accessing it in the view is conventional practice, but it shouldn't be contacting the database at all as far as I can tell